addons/sale_project/controllers/portal.py

```diff
@@ -9,22 +9,22 @@
 
 class SaleProjectCustomerPortal(ProjectCustomerPortal):
 
-    def _task_get_searchbar_groupby(self):
-        values = super()._task_get_searchbar_groupby()
-        values['sale_order'] = {'input': 'sale_order', 'label': _('Sales Order'), 'order': 7}
-        values['sale_line'] = {'input': 'sale_line', 'label': _('Sales Order Item'), 'order': 8}
+    def _task_get_searchbar_groupby(self, milestones_allowed):
+        values = super()._task_get_searchbar_groupby(milestones_allowed)
+        values['sale_order'] = {'input': 'sale_order', 'label': _('Sales Order'), 'order': 8}
+        values['sale_line'] = {'input': 'sale_line', 'label': _('Sales Order Item'), 'order': 9}
```

Comment:
@IT-Ideas I wonder if we could not take into advantage this task to lazy translate the label in `_task_get_searchbar_groupby` and `_task_get_searchbar_inputs` methods. 🤔 

What do you think? 🤔

Reply:
Yes probably, good idea 🙂

Reply:
I know the concept of lazy programming, but in this case, how do you want to proceed ? Do i have to use another function than '_' to translate here ?

Reply:
Yep that's this idea. In fact, when we use the `_` function, the translation will be processed directly. It exists another method called `_lt` to do the translation at the end of the method. By doing that, all the translations will be done at the same time. 🙂